tests/test_dom0_systemd.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+import subprocess
+import unittest
+
+# .spec file shows which units are shipped with rpm
+SYSTEM_UNITS = ["securedrop-logind-override-disable.service"]
+
+USER_UNITS = [
```

Comment:
It would be nice if we could just glob to find these so we don't need to remember to manually add new services/timers to this list.

Reply:
Yeah, I wondered about that, but also I wonder if there is a benefit to being explicit about an exact match of the right number of services.  Fancy us would be able to grab from the spec file and/or the presets files (or, if we had a better directory structure in the workstation repo, we could have `files` have a subdirectory juts for systemd files...), and test that against what's on dom0, instead of globbing twice on dom0. Will think about it